docs/source/installation.rst

```diff
@@ -8,4 +8,10 @@ Install CATS via pip as follows:
 .. code-block:: console
    :caption: *Command to install CATS with pip.*
 
+   $ pip install climate-aware-task-scheduler
```

Comment:
Is this meant to work right now? Because I get:

```console
pip install climate-aware-task-scheduler                                         ─╯
ERROR: Could not find a version that satisfies the requirement climate-aware-task-scheduler (from versions: none)
ERROR: No matching distribution found for climate-aware-task-scheduler
```

Or am I doing something wrong?

Reply:
(I should add, this wasn't using the branch here and assuming we can grab the library via PyPI directly.)

Reply:
It is not yet uploaded to PyPi - it will work when I push the v1 release there

Reply:
OK that makes sense, thanks. Obviously we want to make sure it works ASAP after we officially commit that as the install mechanism, though.